### DISCUSSION TITLE
Does anyone know how use store (Pinia) in boot file?

### DISCUSSION THREAD

**Discussion by Matman94** [quasarframework/quasar#14942] [General - Components / Directives / etc] [answered]
Hi, 
I am missing clear information/example in the documentation on how to use store (pinia) in boot file.

https://quasar.dev/quasar-cli-vite/boot-files

with **Vuex** it was a simply:
```
export default boot(async ({ app, store }) => {
  // something to do
  store.dispatch("device/reloadCache");
})
```
but with **Pinia**?

**Comment by MilosPaunovic** [comment, marked as answer]
This should answer your question:
https://github.com/quasarframework/quasar/discussions/13704#discussioncomment-2962241

**Reply by Matman94** [reply]
Thank You so much!
